xdsl/interactive/app.py

```diff
@@ -126,6 +151,16 @@ def on_mount(self) -> None:
 
         self.query_one("#input_container").border_title = "Input xDSL IR"
         self.query_one("#output_container").border_title = "Output xDSL IR"
+        self.query_one(
+            "#passes_selection_list"
+        ).border_title = "Choose a pass or multiplepasses to be applied."
+
+        # aids in the construction of the seleciton list containing all the passes
+        selections = [(value.name, value) for value in self.list_of_passes]
+        selections.sort()
+        self.passes_selection_list.add_options(  # pyright: ignore[reportUnknownMemberType]
```

Comment:
why the pyright ignore? we generally try to avoid them, especially things like `reportUnknownMemberType`, which might be hiding errors other than those we want to ignore.

Reply:
Ah I see another Textual bug... Could you please file an issue on me to investigate this later? And add a comment above this line? I'll have a look when the next version of Textual comes out.

Reply:
Thank you!